gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -837,13 +958,19 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->input_meta.num_tensors = num_names;
+      } else if (value) {
+        /** Update the properties */
+        if (GST_TF_FW_V0 (priv->fw)) {
+          /* Once configured, it cannot be changed in runtime */
+          g_assert (!prop->input_configured);
+        } else {
+          /** TODO: update the properties */
```

Comment:
Unlike dim/type, this doesn't need re-negotiation. Thus, it appears tractible. However, I don't see any usage cases here.

Reply:
Removed.